projects/oku/programs/oku/module.nix

```diff
@@ -0,0 +1,21 @@
+{
+  config,
+  lib,
+  pkgs,
+  ...
+}:
+let
+  cfg = config.programs.oku;
+in
+{
+  options.programs.oku = {
+    enable = lib.mkEnableOption "oku";
+    package = lib.mkPackageOption pkgs "oku" { };
+  };
+
+  config = lib.mkIf cfg.enable {
+    environment.systemPackages = with pkgs; [
+      cfg.package
+    ];
```

Comment:
```suggestion
    ];
    demo-shell.oku.programs = {
      oku = cfg.package;
    };
```

Reply:
What about https://github.com/ngi-nix/ngipkgs/pull/1406#discussion_r2213062259 ?

Reply:
You're right, but that's when I thought that https://github.com/ngi-nix/ngipkgs/pull/1403 was gonna be merged shortly, which didn't turn out to be the case. Let's keep doing what we're doing until that's resolved.